src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PDFMerger.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.slc.services.steps.getpdf;
+
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import uk.gov.hmcts.reform.slc.services.servicebus.exceptions.PDFMergeException;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.pdfbox.io.MemoryUsageSetting.setupMainMemoryOnly;
+
+public final class PDFMerger {
+
+    private PDFMerger() {
+    }
```

Comment:
i think one of the code style tools require non empty methods. just put in `// utility class constructor` ;)